target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-nbg6817.dts

```diff
@@ -88,6 +107,14 @@
 					drive-strength = <12>;
 				};
 			};
+
+			usbvcc_pins: usbvcc_pins {
+				pins = "gpio14", "gpio15", "gpio16", "gpio17";
+				function = "gpio";
+				drive-strength = <2>;
+				bias-pull-up;
+				output-high;
```

Comment:
If I ask zyxel for the GPL code, is any of you willing to fix this according to the GPL sources?